.github/workflows/linux_single_e2e.yml

```diff
@@ -59,6 +59,43 @@ on:
         default: '{}'
         required: False
 
+  workflow_dispatch:
+    inputs:
+      runner:
+        type: choice
+        options:
+          - '["Linux", "gen12"]'
+          - '["amdgpu"]'
+      image:
+        description: |
+          Use option ending with ":build" for AMDGPU, ":latest" for the rest.
+        type: choice
+        options:
+          - 'ghcr.io/intel/llvm/sycl_ubuntu2204_nightly:latest'
+          - 'ghcr.io/intel/llvm/sycl_ubuntu2204_nightly:build'
+      image_options:
+        description: |
+          Use option with "--device=/dev/kfd" for AMDGPU, without it for the rest.
+        type: choice
+        options:
+          - '-u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN'
+          - '-u 1001 --device=/dev/dri --device=/dev/kfd --privileged --cap-add SYS_ADMIN'
```

Comment:
Why do we want to add `SYS_ADMIN` privs to the docker? Do we already do that?

Reply:
> Why do we want to add `SYS_ADMIN` privs to the docker?

This is still unanswered.
According to oneAPI documentation, this capability is needed for Intel Adviser/Inspector/Vtune tools (https://www.intel.com/content/www/us/en/docs/oneapi-base-toolkit/get-started-guide-linux/2023-2/using-containers-with-the-command-line.html).
AFAIK, our CI doesn't use these tools. Am I missing something?

Reply:
I think we need it for the GPU reset on Intel GPU HW and I wanted to keep options close between these two options here. We might want to lessen those later but I'm focusing on the CI scripts at the moment.

Reply:
Thanks.

> I think we need it for the GPU reset on Intel GPU HW

Should we put this as a comment in the code?

Reply:
Once we spend time to investigate and reduce the options we should document ones that will remain :)